igcollect/url_json.py

```diff
@@ -29,9 +47,20 @@ def main():
 
     template = args.prefix + '.{} {} ' + str(int(time()))
 
-    for key, value in data.items():
-        if key in args.keys or not args.keys:
-            print(template.format(key, value))
+    if args.keys:
```

Comment:
unfortunately this is a breaking change. before it would print everything if no keys were given. now it would not print anything. to restore the previous behaviour we could have another `if not args.keys and not args.jsonpaths` and print all the data

Reply:
I see your point. I considered this a bug as it would print just everything but only for first level and it would print broken metrics in case the value is a list or dictionary.

I will make this backwards compatible.